src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return 'name: ' + this.name
+        + ', chip version: ' + this.version
+        + ', wheels: ' + this.wheels;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(steps = 1) {
+      if (steps < 0) {
+        return this; ;
+      }
+      this.coords.y += steps;
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(steps = 1) {
      if (steps > 0) {
        this.coords.y += steps;
      }
  
      return this;
    },
```
и аналогично с другими